integration-tests/hibernate-orm-panache/src/test/java/io/quarkus/it/panache/PanacheFunctionalityTest.java

```diff
@@ -235,7 +236,118 @@ Person getBug7102(Long id) {
     }
 
     @Test
-    public void testEnhancement27184DeleteDetached() {
+    void testEnhancement27184DeleteDetached() {
         RestAssured.when().get("/test/testEnhancement27184DeleteDetached").then().body(is("OK"));
     }
+
+    /**
+     * This test is disabled in native mode as there is no interaction with the quarkus integration test endpoint.
+     */
+    @Test
+    @Transactional
+    @DisabledOnIntegrationTest
+    void testSimpleEntityProjection() {
+        Person person = new Person();
+        person.name = "1";
+        person.uniqueName = "1";
+        person.persist();
+        PersonName personName = Person.find(" name = ?1", "1")
+                .project(PersonName.class)
+                .firstResult();
+        person.delete();
+        Assertions.assertEquals("1", personName.name);
+    }
+
+    /**
+     * This test is disabled in native mode as there is no interaction with the quarkus integration test endpoint.
+     */
+    @Test
+    @Transactional
+    @DisabledOnIntegrationTest
+    // Test for https://github.com/quarkusio/quarkus/issues/28844
+    void testSimpleEntityProjection2() {
+        Person person = new Person();
+        person.name = "1";
+        person.uniqueName = "1";
+        person.persist();
+        String personName = Person.find("select name from Person2 where name = ?1", "1")
+                .project(String.class)
+                .firstResult();
+        person.delete();
+        Assertions.assertEquals("1", personName);
+    }
+
+    /**
+     * This test is disabled in native mode as there is no interaction with the quarkus integration test endpoint.
+     */
+    @Test
+    @Transactional
+    @DisabledOnIntegrationTest
+    void testNestedEntityProjection_WithQuery() {
+        Person person = new Person();
+        person.name = "2";
+        person.uniqueName = "2";
+        person.address = new Address("street 2");
+        person.persist();
+        PersonDTO personDTO = Person.find(
+                "select uniqueName, name, " +
+                        " new io.quarkus.it.panache.PersonDTO$AddressDTO(address.street)," +
+                        " new io.quarkus.it.panache.PersonDTO$DescriptionDTO(description.size, description.weight)," +
+                        " description.size" +
+                        " from Person2 where name = ?1",
```

Comment:
I'm surprise by this, the project method should built the select clause so why adding it manually here?